.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - stable
+
+script: npm run eslint . npm run test
```

Comment:
eslint is not needed for now on CI. I was planning on implementing this as a git pre-commit hook.

Reply:
Removed